<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovieShowWindow</class>
 <widget class="QMainWindow" name="MovieShowWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="ImageMovie">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="MovieName">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>561</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="DescriptionMovie">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>561</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonReview">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>430</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Оставить рецензию</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="UserGrade">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>44</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="AvgGrade">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonGrade">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Оценить</string>
    </property>
   </widget>
   <widget class="QTableView" name="OtherReview">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>831</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonDelete">
    <property name="geometry">
     <rect>
      <x>548</x>
      <y>430</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить Ревью</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="InputReview">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>831</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonDeleteGrade">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить Оценку</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>856</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
